Assets/Scripts/Layers/LayerTypes/ObjSpawner.cs

```diff
@@ -0,0 +1,93 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Netherlands3D.Twin.Layers.Properties;
+using UnityEngine;
+
+namespace Netherlands3D.Twin.Layers
+{
+    public class ObjSpawner : MonoBehaviour, ILayerWithPropertyData
+    {
+        [Header("Required input")] 
+        [SerializeField] private Material baseMaterial;
+        [SerializeField] private ObjImporter.ObjImporter importerPrefab;
+
+        [Header("Settings")] 
+        [SerializeField] private bool createSubMeshes = false;
+        
+        private ObjPropertyData propertyData = new();
```

Comment:
waarom extra instance?

Reply:
Fields instantieren helpt om het aantal nul checks terug te dringen, want in dit geval is niet verkeerd als er geenwaarde in zit; dan is er minder te serializen. Maar als ik de field instantiatie weglaat, dan moet ik op alle plekken waar ik de field gebruik eraan denken om een null check te doen